src/EventListener/TwoFactorLoginSuccessHandler.php

```diff
@@ -51,34 +45,14 @@
      */
     private $userManager;
 
-    /**
-     * @var array
-     */
-    private $forcedForRoles;
-
-    /**
-     * @var array
-     */
-    private $ipWhiteList;
-
-    /**
-     * @param array $forcedForRole Roles that need to force use 2FA authorization
-     * @param array $ipWhiteList   IP's that will skip 2FA authorization
-     */
     public function __construct(
-        AuthorizationCheckerInterface $authorizationChecker,
         EngineInterface $engine,
         Helper $helper,
-        UserManagerInterface $userManager,
-        array $forcedForRole,
-        array $ipWhiteList
+        UserManagerInterface $userManager
```

Comment:
This is a BC break, so you should target master indeed

Reply:
@greg0ire why this is BC break?)

Reply:
It is BC break because if I was using this class before, now I cannot create objects the same way as before.

Other than that, we could consider not a great deal since this feature was provided by the same user 2-3 days ago and it is not even released yet...

WDYT? BC or not BC?

Reply:
If it is not released, then it is BC :+1:

Reply:
@greg0ire i agree with @jordisala1991 , it's not BC, because it is not released.

Reply:
Then it is BC, because it is not released @aLkRicha xD

Reply:
So hurry up before it isn't anymore :P

Reply:
@greg0ire so change target to master? :D

Reply:
No don't lol

Reply:
@greg0ire ok, so there is all ok 👍